src/components/button/button.spec.tsx

```diff
@@ -102,6 +133,57 @@ describe("Button", () => {
     });
   });
 
+  describe( "Button Minor Padding",() => {
```

Comment:
```suggestion
  describe("Button Minor Padding",() => {
```

Reply:
It might be a bit better to scope this into a describe block for all `isMinor` tests

```
describe("isMinor", () => {
  it.each(isMinorSizesPadding)("when size is %s the padding is %s", (size, padding) => {
        const wrapper = mount(
          <Button isMinor size={size}>Foo</Button>
        );
        assertStyleMatch({padding}, wrapper);
    });
    
    it.each(isMinorColors)("when buttonType is %s", (buttonType, styles) => {
        const wrapper = mount(
          <Button isMinor buttonType={buttonType}>Foo</Button>
        );
        assertStyleMatch({...styles}, wrapper);
     });
     
     it('matches the style for the Button Minor destructive', () => {
        const wrapper = mount(
          <Button isMinor buttonType={buttonType} destructive>Foo</Button>
        )
        assertStyleMatch(
          {
           // assert the relevant destructive styles here
          },
          wrapper
        );
    });

    it('matches the style when disabled', () => {
       const wrapper = mount(
          <Button isMinor buttonType={buttonType} disabled>Foo</Button>
        );
        assertStyleMatch(
          {
            // assert relevant disabled styles here
          },
          wrapper
        );
    });
});
```